### PR TITLE
Create ask page

### PR DESCRIPTION
**Pull request by francium**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -41,6 +42,11 @@ export function view(state, actions) {
         render: ({location}) =>
           Question({id: parseQuestionIdFromPath(location.pathname), data: state.data}),
       }),
+      Route({
+        path: '/ask',
+        render: () =>
```
Comment (francium) [inline src/app.js]: you can make this a one liner, you only split it if the line is too long
`render: () => Ask(),`

File: src/components/ask/ask.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+@import 'src/utils/style-utils.scss';
```
Comment (francium) [inline src/components/ask/ask.scss:1]: Get rid of everything in this file (except the first line import). You don't need any of this for this component.

File: src/components/ask/ask.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import * as html from '@hyperapp/html'
+import classNames from 'classnames'
+
+import styles from './ask.scss'
+
+export default function view() {
+  return html.div(
+    {
+      class: styles.questionPage,
```
Comment (francium) [inline src/components/ask/ask.js]: You can remove this line for now, 
```
return html.div(
  {
  },
  [
    ...
  ]
)
```

**Review by francium — COMMENTED** [review]
Some small changes needed

**Review by francium — COMMENTED** [review]
You got rid of everything for view now, it would be good to have a 'Ask' in there so it still renders something when you go to that page. That's fine for now.

**Comment by francium** [comment]
@khand5 @BJToor Need you to approve this PR

**Review by BJToor — APPROVED** [review]

File: src/app.js [hunk 4]
```diff
@@ -41,6 +42,9 @@ export function view(state, actions) {
         render: ({location}) =>
           Question({id: parseQuestionIdFromPath(location.pathname), data: state.data}),
       }),
+      Route({
+        render: () => Ask(),
```
Comment (francium) [inline src/app.js:47]: You messed up here. You removed too much. You NEED the `path`!

**Pull request merged**